styles/dlc.css

```diff
@@ -604,21 +604,95 @@ li.combatant .roll:hover {
   color: var(--color-text-dark-1);
 }
 
-.trait-button {
+.trait-button1 {
   grid-column: 4;
   color: var(--color-text-dark-1);
   height: var(--charsheet-item-height);
   width: 80px;
 }
 
+.trait-button2 {
+  grid-column: 5;
+  color: var(--color-text-dark-1);
```

Comment:
This is why the colours are wrong when the system is in dark mode. The default `color` is inherited from the `.application` style FVTT sets on the `DLCActorSheet` element as `--color-text-primary`, which maps to `--color-text-dark-1` when the system is in light mode and `--color-light-2` in system dark mode.

We don't need to set a `color`. In my testing we don't need `grid-column` either. I might finally take a crack at the CSS, as you asked for, soon.

Reply:
Be careful of removing grid-column, we likely don't need it on the traits, but I think we do on the aptitudes. The concentration removal uses some of the columns, but not others.

        <div class="**initial-aptitude-entry**">
          <span class="**initial-aptitude-trait**"><strong>{{this}}</strong></span>
          <button 
          class="**initial-aptitude-button2**" 
 
That's 
   .initial-aptitude-trait {
     grid-column: 2;

   .initial-aptitude-button2 {
     grid-column: 6;

If you have some better way to align those in good positions—go for it.

Reply:
ah. yes, I hadn't explored everywhere the classes are used - I'll make sure to do that before I mess with any CSS.